bin/fixmystreet.com/one-off-status-update

```diff
@@ -140,7 +143,15 @@ if ($northants) {
         my $text = $report->{Response_to_Customer};
         # do not add a second response to customer if it's already there.
         if ($text) {
-            my $c = $p->comments->search({text => $text});
+            # sometimes there are extra spaces on the end in the database
+            my $search = { text => { ilike => "$text%" } };
+            # there's quite a few occurrences of the text being corrected that match this
+            if ( $text =~ /.+thank you[.\s]*$/i ) {
+                my $search_text = $text;
+                $search_text =~ s/\sthank you[.\s]*$//;
+                $search = { text => { ilike => "$search_text%" } };
```

Comment:
Do you want a % at the start as well, or are you happy that isn't necessary?

Reply:
I don't especially as there's a few places where they've added updates on the the start of the response which are important.